MetadataExtractor/Formats/Exif/makernotes/OlympusMakernoteDescriptor.cs

```diff
@@ -482,34 +482,21 @@ public OlympusMakernoteDescriptor(OlympusMakernoteDirectory directory)
             if (Directory.GetObject(OlympusMakernoteDirectory.TagWbMode) is not short[] values)
                 return null;
 
-            switch ($"{values[0]} {values[1]}".Trim())
+            return $"{values[0]} {values[1]}".Trim() switch
```

Comment:
This could be a switch on a tuple, to avoid the string allocation and hopefully enable a jump table.

Eg


```suggestion
            return (values[0], values[1]) switch
```

Reply:
Added a dependency to ValueTuple, and updated.